www/.env.sample

```diff
@@ -0,0 +1,2 @@
+REACT_APP_AUTH0_DOMAIN=
```

Comment:
Are we planning to support different auth environments for staging and prod and local?

Reply:
Did not think of that when I wrote this but yes we should so that we could test without affecting deployment :+1: